### DISCUSSION TITLE
deps: v8 ci build node failed

### DISCUSSION THREAD

**Discussion by luyahan** [nodejs/help#4700] [Q&A]
I open a CL on https://chromium-review.googlesource.com/c/v8/v8/+/4293541 to import riscv.gni file
But it build node failed:
log:
```
ERROR at //v8/BUILD.gn:10:1: Can't load input file.
import("//build/config/riscv.gni")
^--------------------------------
Unable to load:
  /b/s/w/ir/cache/builder/node-ci/build/config/riscv.gni
I also checked in the secondary tree for:
  /b/s/w/ir/cache/builder/node-ci/gn/build/config/riscv.gni
See //node/BUILD.gn:142:15: which caused the file to be included.
  configs = [ "//v8:external_config" ]
              ^---------------------
```
riscv.gni had been merged into chromium(https://chromium-review.googlesource.com/c/chromium/src/+/4293162).

What can I do?
Thanks~

**Comment by luyahan** [comment]
Hi @targos 
Do you know what causes it ?

**Reply by targos** [reply]
No, I don't know anything about gn configs and V8's node-ci
